### DISCUSSION TITLE
Episode 138: Sean Parent on Val! (Part 2)

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#29] [Episode]
Episode Link: https://adspthepodcast.com/2023/07/14/Episode-138.html

Feel free to write comments or ask questions 🙂

**Comment by dabrahams** [comment]
Friends, it might be a good idea to tell people _when_ the Val Teams meeting is happening (Tu/Thu 12:30-1 PST) or they're just going to click through and find nobody there :-)

**Reply by codereport** [reply]
:facepalm: That would be helpful indeed. Both the website / podcast episode show notes have been updated. Thanks @dabrahams
